src/main/java/com/pgs/client/dto/TokenDto.java

```diff
@@ -0,0 +1,14 @@
+package com.pgs.client.dto;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class TokenDto {
+    private String access_token;
```

Comment:
We do not use snake_case for non static java field names. You have to configure naming strategy to automatically convert between camelCase and snake_case
https://www.baeldung.com/jackson-advanced-annotations#jsonnaming